packages/sirius-web/backend/sirius-web-services/src/main/java/org/eclipse/sirius/web/services/diagramfilter/ShowButtonDescriptionProvider.java

```diff
@@ -0,0 +1,74 @@
+package org.eclipse.sirius.web.services.diagramfilter;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.collaborative.diagrams.dto.HideDiagramElementInput;
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.core.api.IObjectService;
+import org.eclipse.sirius.components.diagrams.Diagram;
+import org.eclipse.sirius.components.forms.ButtonStyle;
+import org.eclipse.sirius.components.forms.WidgetIdProvider;
+import org.eclipse.sirius.components.forms.description.ButtonDescription;
+import org.eclipse.sirius.components.representations.VariableManager;
+import org.eclipse.sirius.web.services.diagramfilter.api.IDiagramFilterActionContributionProvider;
+import org.springframework.stereotype.Service;
+
+/**
+ * Provides the description of the show button for the diagram filter's split button.
+ *
+ * @author gdaniel
+ */
+@Service
+public class ShowButtonDescriptionProvider implements IDiagramFilterActionContributionProvider {
+
+    private IObjectService objectService;
+
+    private DiagramFilterHelper diagramFilterHelper;
+
+    public ShowButtonDescriptionProvider(IObjectService objectService, DiagramFilterHelper diagramFilterHelper) {
+        this.objectService = objectService;
+        this.diagramFilterHelper = diagramFilterHelper;
+    }
+
+    @Override
+    public ButtonDescription getButtonDescription() {
+        return ButtonDescription.newButtonDescription("diagram-filter/split-button/show")
+        .idProvider(new WidgetIdProvider())
+        .targetObjectIdProvider(variableManager -> variableManager.get(VariableManager.SELF, Object.class).map(this.objectService::getId).orElse(null))
+        .labelProvider(variableManager -> "Show")
+        .iconURLProvider(variableManager -> List.of())
+        .isReadOnlyProvider(variableManager -> false)
+        .buttonLabelProvider(variableManager -> "Show")
+        .imageURLProvider(variableManager -> "/icons/full/obj16/ShowTool.svg")
+        .pushButtonHandler(variableManager -> {
+            var diagram = variableManager.get("diagram", Diagram.class).get();
+            var nodeIds = this.diagramFilterHelper.getSelectedElementIds(variableManager);
+            var editingContext = variableManager.get("editingContext", IEditingContext.class).get();
+            return this.diagramFilterHelper.sendDiagramEvent(variableManager, new HideDiagramElementInput(UUID.randomUUID(), editingContext.getId(), diagram.getId(), nodeIds, false));
+        })
+        .diagnosticsProvider(variableManager -> List.of())
+        .kindProvider(variableManager -> "")
+        .messageProvider(variableManager -> "")
+        .styleProvider(variableManager -> ButtonStyle.newButtonStyle()
+                .backgroundColor("#ffffff")
+                .foregroundColor("#000000")
+                .build()
+        )
+        .helpTextProvider(variableManager -> "Show selected elements")
+        .build();
```

Comment:
Invalid formatting, it has not the same formatting as other buttons

Reply:
done